File: src/learning_gem5/jiajiapart2/jiajia_object.cc
#include "jiajia_object.hh"
#include <iostream>

namespace gem5
{

JiajiaObject::JiajiaObject(const JiajiaObjectParams &params) :
    SimObject(params)
{
    std::cout << "Hello World! jiajia From a SimObject!" << std::endl;
}

} // namespace gem5

File: src/learning_gem5/jiajiapart2/JiajiaObject.py
from m5.params import *
from m5.SimObject import SimObject


class JiajiaObject(SimObject):
    type = "JiajiaObject"
    cxx_header = "learning_gem5/jiajiapart2/jiajia_object.hh"
    cxx_class = "gem5::JiajiaObject"
